common/app/components/Nav/index.js

```diff
@@ -1 +1 @@
-export default from './Nav.jsx';
```

Comment:
This is valid syntax:

see https://github.com/leebyron/ecmascript-export-default-from